sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -168,6 +169,17 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
   async getECertInformationToBeSent(
     offeringIntensity: OfferingIntensity,
   ): Promise<DisbursementSchedule[]> {
+    let possibleRestrictionActions: RestrictionActionType[];
+    if (offeringIntensity === OfferingIntensity.fullTime) {
+      possibleRestrictionActions = [
+        RestrictionActionType.StopFullTimeDisbursement,
+      ];
+    }
+    if (offeringIntensity === OfferingIntensity.partTime) {
```

Comment:
We can do this in one inline condition avoiding 2 if conditions.
`possibleRestrictionActions = (offeringIntensity === OfferingIntensity.fullTime) ? [RestrictionActionType.StopFullTimeDisbursement] : [RestrictionActionType.StopPartTimeDisbursement]`